init-tools.msbuild

```diff
@@ -11,15 +11,6 @@
     <PackageReference Include="microsoft.dotnet.buildtools">
       <Version>$(BuildToolsPackageVersion)</Version>
     </PackageReference>
-    <PackageReference Include="microsoft.dotnet.buildtools.coreclr">
```

Comment:
Please don't remove this package. Without it, the debugger will most likely break.

Reply:
Are you referring to the SOS debugger?  
Can you explain, why the debugger would fail?  
Why doesn't the build error?  Silent breaks worry me.

Reply:
No just SOS, but most managed debugging APIs. Although I'll have to take a further look if it actually breaks. The package contains the following:

- `DacTableGen`
- `DiaLib`
- `InjectResource`
- `GenClrDebugResource`
- `PbdTypeMatch`

The `DacTableGen` is now built from source, but or Windows debugging you can see in https://github.com/dotnet/coreclr/blob/master/src/dlls/mscoree/coreclr/CMakeLists.txt#L172-L182 we use `InjectResource` and `GenClrDebugResource` to generate metadata we need to be able to debug. As if it's used in some way and it's not breaking the build I'll have to confirm manually. It's definitely something that should break the build.

If you can, please add the package back. I'd like to build this from source as we are the only consumers and then we can just use if from the build itself and delete this package.

Reply:
Done